src/ElementFinder.php

```diff
@@ -214,7 +214,18 @@ private function setData(string $data): self
 
         if (static::DOCUMENT_HTML === $this->type) {
             $data = StringHelper::safeEncodeStr($data);
-            $data = mb_convert_encoding($data, 'HTML-ENTITIES', 'UTF-8');
+
```

Comment:
I think we can use `htmlentities`
![image](https://github.com/user-attachments/assets/f05ca4a5-4a42-493f-a899-3e0b15fbbe6c)
Have you tried it?

Reply:
When using only htmlentities, some special characters will remain encoded, as shown in the screenshot above. To decode them, you need to use htmlspecialchars_decode, and also use mb_encode_numericentity to properly handle characters in UTF-8 encoding.

Reply:
Could you give an example when special symbols are not encoded? I want to check this strings on both converters current and a new one.

Reply:
I meant that if you use only `htmlentities`, then the characters `"'&<>` will remain encoded and you will need to additionally use `html_entity_decode` to convert them.

In addition, if you use only `htmlentities` and `html_entity_decode`, then in the tests I added you will see that in the second case, when "broken" characters are used, `htmlentities` successfully removes them, but in the first test case, instead of "Текст текст" it will output Ð¢ÐµÐºÑ... and so on.
![image](https://github.com/user-attachments/assets/108f5485-9e42-493f-b08d-59d0bf94c77c)